client-app/core/types/notifications.ts

```diff
@@ -0,0 +1 @@
+export type UnreadCountType = { totalCount: number };
```

Comment:
notifications is a separate module. Move this type to the appropriate folder. Or use Pick<..., totalCount> in place